ui/src/containers/Layout.svelte

```diff
@@ -3,123 +3,198 @@
   import { Navigate, navigateTo } from "svelte-router-spa";
   import { userSession, userName, isLoggedIn } from "../stores.js";
   import { scale } from "svelte/transition";
+  import { onMount } from "svelte";
 
   export let currentRoute;
 
-  let scanActive = false, settingActive = false, signOutActive = false;
-  
+  let scanActive = false,
+    settingActive = false,
+    signOutActive = false;
+
   let menu = false;
 
   const params = {};
 
   const signOut = () => userSession.logout();
 
+  let latestBuildDaysAgo = "";
+  let latestBuildNumber = "";
+
+  onMount(async () => {
+    const deploymentUrl =
+      "https://api.github.com/repos/SSWConsulting/SSW.CodeAuditor/deployments?environment=production&task=deploy&per_page=1";
```

Comment:
This API string should be put in the .env file

Reply:
Add to rollup file after .env and also as an environment variable/secret in GitHub. Then update workflows file (staging-build.yml & main.yml)